services/fonts.js

```diff
@@ -0,0 +1,225 @@
+'use strict';
+
+var xtend = require('xtend');
+var v = require('./service-helpers/validator');
+var pick = require('./service-helpers/pick');
+var createServiceFactory = require('./service-helpers/create-service-factory');
+
+/**
+ * Fonts API service.
+ *
+ * Learn more about this service and its responses in
+ * [the HTTP service documentation](https://docs.mapbox.com/api/maps/fonts/).
+ */
+var Fonts = {};
+
+/**
+ * Get a font glyph range.
+ *
+ * See [the corresponding HTTP service documentation](https://docs.mapbox.com/api/maps/fonts/#retrieve-font-glyph-ranges).
+ *
+ * @param {Object} config
+ * @param {string|Array<string>} config.fonts - An array of font names.
+ * @param {number} config.start - Character code of the starting glyph.
+ * @param {number} config.end - Character code of the last glyph,
+ *   typically equivalent to`config.start + 255`.
+ * @param {string} [config.ownerId]
+ * @return {MapiRequest}
+ *
+ * @example
+ * fontsClient.getFontGlyphRange({
+ *   fonts: 'Arial Unicode',
+ *   start: 0,
+ *   end: 255
+ * })
+ *   .send()
+ *   .then(response => {
+ *     const glyph = response.body;
+ *   });
+ */
+Fonts.getFontGlyphRange = function(config) {
+  v.assertShape({
+    fonts: v.required(v.oneOfType(v.string, v.arrayOf(v.string))),
+    start: v.required(v.number),
+    end: v.required(v.number),
```

Comment:
Since we know that `end` is always `start + 255`, we could make this method a little more ergonomic by only requiring the `start` parameter. Then, we'd just add `255` to it to come up with the correct fileName:

```js
var fileName = config.start + '-' + (config.start + 255) + '.pbf';
```

Reply:
Must end always equal start+255 or could users still request a larger slice?

Reply:
Yup, `end` must always be `start + 255`